collections/sentinel-3-l1b-slstr.yaml

```diff
@@ -89,12 +89,14 @@ CubeDimensions:
     extent:
       - -180
       - 180
+    reference_system: 4326
   y:
     type: spatial
     axis: y
     extent:
       - -85
       - 85
+    reference_system: 4326
   t:
     type: temporal
     extent:
```

Comment:
Here revisit time is
<html>
<body>
<!--StartFragment-->

Revisit time | < 0.9 days at the equator with 2 satellites
-- | --


<!--EndFragment-->
</body>
</html>

Reply:
Any idea how to write this correctly? I think the ISO8601 duration doesn't foresees dates with "<". 
https://github.com/stac-extensions/datacube#temporal-dimension-object

Reply:
Well the "correct" value is probably `null`, but in a practical sense `P1D` would probably make most sense. Because openEO anyway typically works with days and not specific orbits.

Reply:
I just now saw that those 3 collections all had `P5D` set. Corrected everywhere to either `P1D` or `P2D`.